runtime/parser/statement.go

```diff
@@ -207,7 +205,9 @@ func parseReturnStatement(p *parser) (*ast.ReturnStatement, error) {
 	endPosition := tokenRange.EndPos
 	p.next()
 
-	sawNewLine := p.skipSpaceAndComments(false)
```

Comment:
this is the only place where we skip whitespace excluding newlines, so it's simpler to just inline the call here instead.